etsin_finder/finder.py

```diff
@@ -71,8 +71,11 @@ def add_restful_resources(app):
     """
     api = Api(app)
     from etsin_finder.resources import Contact, Dataset, User, Session, Files, Download
+    from etsin_finder.qvain_light_resources import ProjectFiles, FileDirectory
```

Comment:
ProjectFiles is for getting the project root directory, FileDirectory is for getting the contents of a single directory. Both are in their own file, ```qvain_light_resources```.